tools/src/commands/GenerateDocsAPIData.ts

```diff
@@ -26,7 +26,17 @@ const PACKAGES_MAPPING: Record<string, CommandAdditionalParams> = {
   'expo-apple-authentication': ['index.ts'],
   'expo-application': ['Application.ts'],
   'expo-audio': [['Audio.ts', 'Audio.types.ts'], 'expo-av'],
-  'expo-auth-session': ['AuthSession.ts'],
+  'expo-auth-session': [
+    [
+      'AuthRequest.ts',
+      'AuthSession.ts',
+      'Discovery.ts',
+      'Errors.ts',
+      'Fetch.ts',
+      'TokenRequest.ts',
+      'providers/Provider.types.ts',
+    ],
+  ],
```

Comment:
What do you think if we make the index file instead?

Reply:
Will make a follow up PR with that change, if we decided to make those changes within this changes it would be harder to backport the fixes to SDK 51, if we do not cherry pick `index.ts` creation to `sdk-51` branch.